js/scorm/wrapper.js

```diff
@@ -336,7 +336,12 @@ define([
       this.lmsConnected = false;
 
       if (!this.scorm.quit()) {
-        this.handleError(new ScormError(CLIENT_COULD_NOT_FINISH));
+        const errorCode = this.scorm.debug.getCode();
+        this.handleError(new ScormError(CLIENT_COULD_NOT_FINISH, {
+          errorCode,
```

Comment:
Where does this error info come out?

[Message definition](https://github.com/adaptlearning/adapt-contrib-spoor/blob/7173b1269cc84a4ebe10b8be6c8b6398501da94b/js/scorm/error.js#L30)
[Error handler](https://github.com/adaptlearning/adapt-contrib-spoor/blob/7173b1269cc84a4ebe10b8be6c8b6398501da94b/js/scorm/wrapper.js#L498-L532)

Reply:
Ha! Good point. I'll sort...